src/win/fs.c

```diff
@@ -528,7 +528,7 @@ void fs__close(uv_fs_t* req) {
 
   VERIFY_FD(fd, req);
 
-  result = _close(fd);
+  result = uv__is_stdio_handle((HANDLE)fd) ? 0 : _close(fd);
```

Comment:
This doesn't work. You can't cast between HANDLEs and file descriptors.
This path closes a file descriptor; msvcrt maintains an internal table that maps FDs to HANDLEs.
So here you want to either use unix logic (`fd <= 2`) or translate the fd to a handle (with `_get_osfhandle()`).